roles/sat6repro/tasks/main.yml

```diff
@@ -71,20 +82,11 @@
   command: katello-installer
   when: satelliteversion == 6.1
 - name: run Satellite 6.2 installer
-  command: satellite-installer --scenario satellite
+  command: satellite-installer --scenario satellite --foreman-proxy-dns false --foreman-proxy-dhcp false
```

Comment:
what is the need to specify `--foreman-proxy-dns false --foreman-proxy-dhcp false` explicitly?